backend/db/LoggedDatabaseClient.ts

```
@@ -0,0 +1,287 @@
+import { pino } from 'pino';
+import { DatabaseClient, RedisSortedSetItem, FeedItem, Quote, Compressed } from '../types/index.js';
+import { LogContext, ReadOptions, /* ScanOptions, RangeOptions */ } from './loggingTypes.js'; // Import LogContext and ReadOptions
+
+// Interface for logging context passed to methods
+export interface LogContext {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'LogContext'. Consider to delete it or rename it.

'LogContext' is defined here:


(lint/suspicious/noRedeclare)